pkg/scheduler/framework/frameworkutils.go

```diff
@@ -135,30 +139,32 @@ func crossReferencePickedCustersAndObsoleteBindings(
 
 		// The binding's target cluster is picked again in the current run; yet the binding
 		// is originally created/updated in accordance with an out-of-date scheduling policy.
+		// Add the binding to the toPatch list. We will simply keep the binding's state as
+		// it could be "scheduled" or "bound".
+		toPatch = append(toPatch, patchBinding(binding, binding.Spec.State, scored, policy))
```

Comment:
Hi Ryan! A concern is that, since we are using patch rather than update, this might overwrite a binding from bound back to scheduled under unfortunate circumstances (scheduler sees scheduled, but rollout has marked it as bound)